component/Buttons/button.html

```diff
@@ -94,7 +99,7 @@ <h2>Example of Primary Buttons</h2>
 						>
 						</iframe>
 					</div>
-					<h2>Example of Link Button</h2>
+					<h2 class="h2">Example of Link Button</h2>
```

Comment:
instead of doing `<h2 class="h2">text</h2>`
directly set the size of h2 the you can use `<h2>text</h2>`

Reply:
I have not done that because in some places the simple h2 functionality is used, putting the h2 class functionality overrides it.